AdminWebsite/AdminWebsite/Helper/AdministratorRoleClaimsHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AdminWebsite.Security;
+using AdminWebsite.Services.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+
+namespace AdminWebsite.Helper
+{
+    public class AdministratorRoleClaimsHelper
```

Comment:
Do we need the `Helper` in the name or does the name excluding `Helper` explain properly what it does?

Reply:
not sure, without helper dont know what it does...any suggestions ?